tools/cockpit.spec

```diff
@@ -347,8 +347,8 @@ Cockpit support for reading PCP metrics and loading PCP archives.
 %package dashboard
 Summary: Cockpit SSH remoting and dashboard
 Requires: libssh >= %{libssh_version}
-Requires: cockpit-ws = %{version}-%{release}
-Provides: cockpit-ssh = %{version}-%{release}
+Requires: cockpit-bridge > 134
```

Comment:
But having a higher dependency in the downstream packaging  than in the upstream code is completely arbitrary, as we don't have any downstream changes which would change that relationship. Either there is a justification for ≥ 134 and then manifest.json as the one source of truth should reflect that, or there isn't, and then we should just use `%required_base` here without bumping (then it'll come out as `>= 132.x`).